lampod-cli/src/main.rs

```diff
@@ -155,7 +155,9 @@ fn run(args: LampoCliArgs) -> error::Result<()> {
         std::process::exit(0);
     })?;
 
-    let workder = lampod.listen().unwrap();
+    let workder = lampod.listen().map_err(|err| {
+        return error::anyhow!("{err}");
+    })?;
```

Comment:
```suggestion
    let workder = lampod.listen()?;
```

Please, do not apply monkey change

Reply:
Thanks, changed